src/docfx.website.themes/default/partials/head.tmpl.partial

```diff
@@ -14,6 +14,7 @@
   <link rel="stylesheet" href="{{_rel}}styles/main.css">
   <meta property="docfx:navrel" content="{{_navRel}}">
   <meta property="docfx:tocrel" content="{{_tocRel}}">
+  {{#_noindex}}<meta name="searchOption" content="noindex">{{/_noindex}}
```

Comment:
naming: Options, as you implements, content can be `"noindex, nofollow"` which contains multiple options?